test/e2e/pivoting.go

```diff
@@ -262,7 +262,7 @@ type RemoveIronicInput struct {
 
 func removeIronic(ctx context.Context, inputGetter func() RemoveIronicInput) {
 	input := inputGetter()
-	if input.IsDeployment == true {
+	if input.IsDeployment {
```

Comment:
golangci-lint: 
```
 test/e2e/pivoting.go:265:5: S1002: should omit comparison to bool constant, can be simplified to `input.IsDeployment` (gosimple)
	if input.IsDeployment == true { 
```

Reply:
Indeed, this is one of the benefits of using booleans directly instead of strings. :+1: